modules/documents/spec/controllers/documents_controller_spec.rb

```diff
@@ -67,8 +67,14 @@
       get :new, params: { project_id: project.id }
     end
 
+    it "returns a successful response" do
+      expect(response).to be_successful
+    end
+
     it "show the new document form" do
-      expect(response).to render_template(partial: "documents/_form")
+      expect(response.body).to include("Category")
+      expect(response.body).to include("Title")
+      expect(response.body).to include("Description")
```

Comment:
I know we have `render_views` enabled, but I don't think Controller specs should really be concerned with testing rendering.

```suggestion
      expect(response).to render_template("new")
```

Reply:
Sounds fair. I am adjusting it to follow the same standard as other tests in the file